### PR TITLE
Patch logout

### PR DESCRIPTION
**Pull request by JosueMolinaMorales**
Tries to complete [8845](https://app.shortcut.com/clarkcan/story/8845/logout-not-working-in-prod)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,6 +38,7 @@
     "ng2-charts": "^2.3.0",
     "ng2-ckeditor": "^1.3.6",
     "ngx-cookie": "^5.0.2",
```
Comment (mfranz1) [inline package.json:40]:
```suggestion
```

File: src/app/core/auth.service.ts [hunk 2]
```diff
@@ -6,7 +6,8 @@ import {
 } from '@angular/common/http';
 import { environment } from '@env/environment';
 import { Observable, BehaviorSubject, throwError, of } from 'rxjs';
-import { CookieService } from 'ngx-cookie';
+// import { CookieService } from 'ngx-cookie';
```
Comment (mfranz1) [inline src/app/core/auth.service.ts:9]: delete this

File: src/app/core/auth.service.ts [hunk 3]
```diff
@@ -97,14 +98,22 @@ export class AuthService {
   private endSession() {
     this.user = undefined;
     this.openIdToken = null;
-    this.cookies.remove('presence');
+
+    const domain = environment.production ? 'clark.center' : 'localhost';
+    this.cookies.delete('presence', '/', domain, false);
```
Comment (mfranz1) [inline src/app/core/auth.service.ts:103]:
```suggestion
    this.cookie.delete('presence', '/', domain, false, 'Lax');
```

**Review by mfranz1 — CHANGES_REQUESTED** [review]

File: src/app/core/auth.service.ts (hunk 2 above)
Comment (mfranz1) [reply src/app/core/auth.service.ts:9]: nevermind

**Review by mfranz1 — DISMISSED** [review]
Looks good, give this a patch before committing to hotfix

**Review by mfranz1 — APPROVED** [review]

**Review by EVogel1999 — APPROVED** [review]

**Pull request merged**